src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.html

```diff
@@ -61,14 +61,20 @@ <h4 *ngIf="!fileUploadExercise.id" id="jhi-file-upload-exercise-heading-create"
             ></jhi-date-time-picker>
             <span *ngIf="fileUploadExercise.assessmentDueDateError" class="invalid-feedback">{{ 'artemisApp.exercise.assessmentDueDateError' | artemisTranslate }}</span>
         </div>
-        <div class="form-group">
+        <div class="form-group" *ngIf="!isExamMode">
             <label class="form-control-label">{{
                 'artemisApp.exercise.includedInOverallScore' + (fileUploadExercise.course ? 'Course' : 'Exam') + 'Label' | artemisTranslate
```

Comment:
Since we check for `isExamMode` above, the label translation here and below can be simplified to directly `includedInOverallScoreCourseLabel` (and `includedInOverallScoreExamLabel` below).